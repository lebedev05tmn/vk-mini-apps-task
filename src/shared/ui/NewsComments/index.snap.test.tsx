import { NewsComments } from "./index";

const ReactTestRenderer = require("react-test-renderer");

describe("NewsComment render", () => {
  test("Should NewsComment with children", () => {
    const tree = ReactTestRenderer.create(
      <NewsComments>
        <div>hello world</div>
      </NewsComments>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
