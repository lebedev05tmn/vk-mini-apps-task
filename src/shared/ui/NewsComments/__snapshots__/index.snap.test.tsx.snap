// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewsComment render Should NewsComment with children 1`] = `
<div
  className="vkuiCard"
>
  <div>
    hello world
  </div>
</div>
`;
